i18n-util/src/UrlUtil.js

```diff
@@ -1,4 +1,12 @@
 
+
+const getUrlParameter = (name) => {
+  name = name.replace(/[\\[]/, '\\[').replace(/[\]]/, '\\]');
+  var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+  var results = regex.exec(window.location.hash);
+  return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
```

Comment:
To ensure we check for the return length, consider changing this to:

```suggestion
  return (results && results.length > 1) ? decodeURIComponent(results[1].replace(/\+/g, ' ')) : null;
```